File: projects/project10/Java/WalkingBird.java
/* WalkingBird.java provides a class for the WalkingBird types
 *
 * Created by: Ethan Clark Project10
 * Date: April 25, 2016
 */

public class WalkingBird extends Bird
{

	/*******************************************************
	* Constructor for WalkingBird class                    *
	* Calls the name constuctor for the Bird parent class  *
	*******************************************************/
	public WalkingBird(String name)
	{
		super(name);
	}

	/*****************************************************************
	* movement() returns the movement type for the WalkingBird class *
	* Receive: NONE                                                  *
	* Return: "walked"                                               *
	*****************************************************************/
	public String movement()
	{
		return "walked";
	}

}

File: projects/project10/Ruby/WalkingBird.rb
# WalkingBird.rb | Defines the WalkingBird class which inherits attributes and methods
#   from the Bird superclass.
#
# Created by: Ethan Clark Project10
# Date: April 26, 2016
####################################################

require './Bird.rb'

class WalkingBird < Bird

	###########################################################
	# movement returns the move that a WalkingBird type does  #
	# Recieve: NONE                                           #
	# Return: walked, a String                                #
	###########################################################
	def movement
		"walked"
	end

end
